Documentation/kubernetes/install.rst

```diff
@@ -16,6 +16,80 @@ the :ref:`k8scompatibility` section for kubernetes API version compatibility.
 
 .. _admin_mount_bpffs:
 
+Running Kubernetes with CRD Validation (Recommended)
+====================================================
+
+Custom Resource Validation was introduced in Kubernetes since version ``1.8.0``.
+This is still considered an alpha feature in Kubernetes ``1.8.0`` but since
+``v1.0.0-rc3``, Cilium creates, or updates in case it exists, the Cilium Network
```

Comment:
"Cilium Network Policy (CNP)"

This clarifies the abbreviation used later in the section.

Reply:
Done